features/steps/ccx_data_engineering_service.py

```diff
@@ -45,3 +45,26 @@ def start_ccx_upgrades_data_eng(context, port):
     assert popen is not None
     time.sleep(2)
     context.add_cleanup(popen.terminate)
+
+
+@given("The mock RHOBS Service is running on port {port:d}")
+def start_RHOBS_mock_service(context, port):
+    """Run RHOBS service mock for a test and prepare its stop."""
+    params = ["uvicorn", "mocks.rhobs.rhobs_service:app", "--port", str(port)]
+
+    f = open(f"logs/ccx-upgrades-data-eng/{context.scenario}.log", "w")
+    popen = subprocess.Popen(params, stdout=f, stderr=f)
+    assert popen is not None
+    time.sleep(0.5)
```

Comment:
prob. better to active check for port to be accessible?

Reply:
true, let me change that